product_subscription_web_access/controllers/main.py

```diff
@@ -4,7 +4,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from openerp.addons.website_product_subscription.controllers.subscribe import (
-    SubscribeController
+    SubscribeController,
 )
```

Comment:
why using parenthesis and coma for a simple import... #noqa could apply here

Reply:
Kind of agree but I'll leave black settle this.